.env.example

```diff
@@ -3,6 +3,13 @@ LOGGING_LEVEL=info
 
 WEB_CONCURRENCY=1
 
+V8_PROFILER_ENABLED=
```

Comment:
I'd also recommend to prefix env variables with `GAMBIT_`, in case we wanted to expose them through normal env vars. So they don't conflict with other projects that may want to use, for example, AWS bucket.